bindings/pyroot_experimental/PyROOT/src/PyzCppHelpers.cxx

```diff
@@ -0,0 +1,29 @@
+// Author: Danilo Piparo CERN  08/2018
+
+/*************************************************************************
+ * Copyright (C) 1995-2018, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#include "PyzCppHelpers.hxx"
+
+#include "CPyCppyy.h"
+#include "CPPInstance.h"
+#include "TClass.h"
+
+PyObject *CallPyObjMethod(PyObject *obj, const char *meth, PyObject *arg1)
```

Comment:
As commented [here](https://sft.its.cern.ch/jira/browse/ROOT-9638), to be seen if this method can be exposed by Cppyy.

Reply:
Let me then comment about this jira item in the code and make clear that this is a temporary workaround.

Reply:
OK